payments/tests/test_payments_mutations.py

```diff
@@ -1340,3 +1351,133 @@ def test_confirm_payment_invalid_status(old_schema_api_client, status):
     mock_call.assert_not_called()
     assert_in_errors("The order is not valid anymore", executed)
     assert_in_errors(status.label, executed)
+
+
+APPROVE_ORDER_MUTATION = """
```

Comment:
Would it make sense to create own test file for these e.g. `test_payments_mutations_approve_order.py` since the file is already huge?

Reply:
On one hand I agree that it would be nice to split them somehow to avoid long extremely long files.
On the other, we don't have any other test file split by behaviour (?), but it could be a good start with the right approach.

Reply:
yeah, I think it would be a step to the right direction.

Reply:
Pushed 👍